docs/reference/edot-collector/components.md

```diff
@@ -20,6 +20,11 @@ The {{edot}} (EDOT) Collector includes embedded Collector components from the [O
 
 The components included in the EDOT Collector are categorized into **[Core]** and **[Extended]** components. The following table describes the current components included in the EDOT Collector, their source, and support status.
 
+% The following table is automatically generated from the EDOT Collector source code.
+% Automation is handled by /docs/scripts/update-docs/update-components-docs.py, which
```

Comment:
Why is this using a different approach, than it is used for generating https://github.com/elastic/elastic-agent/tree/main/internal/pkg/otel#components?

Does  /docs/scripts/update-docs/update-components-docs.py and https://github.com/elastic/elastic-agent/blob/main/dev-tools/mage/otel/deps.go generate the same output? Does special case handling (like https://github.com/elastic/elastic-agent/pull/10143) be updated in multiple places?